src/components/2_molecules/navigation/primaryNav.js

```diff
@@ -30,25 +23,53 @@ class MainSiteNav extends React.Component {
     return false;
   }
 
+  // do this on initial page render/re-render
   componentDidMount() {
-    const subMenus = document.getElementById("sub-menus");
-    const otherMenus = document.getElementsByClassName("sub-menu");
-    const header = document.getElementById("header");
+    const subMenus = document.getElementsByClassName("c-sub-menu");
+    const header = document.getElementById("menu-header-primary");
 
     const hideAll = () => {
```

Comment:
This shouldn't be defined inside a componentDidMount. As is it's defined twice. It can be abstracted out of the component.

Reply:
Or if it needs something inside the component (state or props) then it can be an instance method.